src/librustdoc/doctest.rs

```diff
@@ -803,6 +805,22 @@ fn run_test(
     let duration = instant.elapsed();
     if doctest.no_run {
         return (duration, Ok(()));
+    } else if doctest.langstr.should_panic
```

Comment:
Should we leave a FIXME somewhere that we (probably) don't handle `-Cpanic=immediate-abort` correctly (very recently added)? As I've PM'ed you, I actually get *illegal instruction* for panics under this strategy and I don't know if that's intentional or not.

Reply:
I believe that's due to llvm lowering `core:: intrinsics::abort` to ub2 which gets translated to an illegal instruction

https://github.com/rust-lang/rust/issues/81895